docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -98,13 +98,18 @@ def downstream_asset(upstream_asset):
     return upstream_asset + [4]
 
 
+s3_resource = S3Resource()
```

Comment:
since it's just one line, should we move this to inline?

Reply:
yes 👍